### ISSUE TITLE
Update screenspace renderables to increase ease of use

### ISSUE THREAD

**Issue by WeirdRubberDuck** [OpenSpace/OpenSpace#2250]
Want to make it easier to put screenspace images for example at a corner of the screen. Some concrete things to update: 

- Remove Z/radius property (that currently affect the visual size of the renderable) and instead order based on order in the list 
- Add option to reorder screen space renderables list similarly to layers (consequence of above) - OBS includes WebUI update
- Add dropdowns to set anchor/align points. 9 points for the image and window, rerspectively: BottomRight, BottomLeft, BottomCenter, LeftCenter, Center, RightCenter, etc.. (see image below) 
- Separate cartesian and radiusAzimuthElevation (RAE) logic. Use RAE automatically if using weird (non-planar) proejctions, and cartiesuan for planar projections
- Add trigger property to copy cartiesian position to RAE (using the current conversion)

To place an image X pixels from edge of window
- Add pixel padding property to the ScreenSpaceImages 
- helper function to compute cartesian position from window coordinates (e.g.  `openspace.screenspace.createRelativePosition("BottomRight", 50, 50)`)

To figure out: 
- Do we still want the "face Camera" checkbox?

Note: No changes to the current coordinate systems 

![image](https://user-images.githubusercontent.com/8808894/192497758-307579b6-e8a8-4405-a2f7-5bdb54786303.png)

**Comment by micahnyc** [comment]
It occurred to me today that maybe we should have a screen space translation rather then screenspace renderable. Maybe this doesn’t make sense b/c of lighting and too many renderbales assuming they are part of the normal scene graph

**Comment by alexanderbock** [comment]
As we discussed in the meeting afterwards, we will go the way of creating a new ScreenSpace type that can take an arbitrary Renderable subclass and render it from a fixed camera instead to get this behavior

**Comment by buddy326** [comment]
Hi, Im a student in software engineering and was hoping to work on this issue as part of a lab. Would I please be able to take this?

**Comment by alexanderbock** [comment]
Absolutely!  If you have some more questions about it, the fastest way would be to use the public Slack (https://join.slack.com/t/openspacesupport/shared_invite/zt-24uhn3wvo-gCGHgjg2m9tHzKUEb_FyMQ) and then ping me or one of the other developers.  You'll have to implement the course work, but if you have some questions surrounding the integration into OpenSpace, we're happy to give some pointers!

**Comment by Arohdin** [comment]
Radius control is actually nice to have when rendering in stereoscopic as it makes it possible to control how close/far away the screenspacerenderable is to the viewer. Sometimes you may want the screenspace image to "float" in the air, but sometimes you want it to feel like it "isn't floating" (example, when you render `ScreenSpaceInsetBlackout` shape).
